lib/ZnapZend/Config.pm

```diff
@@ -47,7 +47,9 @@ has backupSets => sub { [] };
 
 ### private functions ###
 my $splitHostDataSet = sub {
```

Comment:
Set host bate

Reply:
@Ariessanea1 : what's with the spam across many repos?